mlir/lib/Conversion/MemRefToEmitC/MemRefToEmitC.cpp

```diff
@@ -97,6 +98,31 @@ Type convertMemRefType(MemRefType opTy, const TypeConverter *typeConverter) {
   return resultTy;
 }
 
+Value calculateMemrefTotalSizeBytes(Location loc, MemRefType memrefType,
```

Comment:
```suggestion
static Value calculateMemrefTotalSizeBytes(Location loc, MemRefType memrefType,
```
I don't think this gets used outside this TU, right?